### ISSUE TITLE
debug logging filling up log with un-useful information

### ISSUE THREAD

**Issue by finity69x2** [custom-components/authenticated#37]
**Version of the custom_component**

0.4.2

**Describe the bug**

the sensor causes a large number of logbook entries because the component sets the log level to "debug".

**log**

```
2019-04-30 01:39:55 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/websocket to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/card-tools.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/slider-entity-row.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/radar-forecast.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/tracker-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/entity-attributes-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/vertical-stack-in-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/dark-sky-weather-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/secondaryinfo-entity-row.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/atomic-calendar.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/tiles-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/monster-card.js to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:39:58.788Z to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:39:58.787Z to 192.168.1.124 (auth: True)
2019-04-30 01:39:57 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:39:58.789Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:00 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:39:58.788Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:00 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:39:58.789Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:00 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:39:58.787Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:01 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:40:01.474Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:01 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:40:01.487Z to 192.168.1.124 (auth: True)
2019-04-30 01:40:01 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:40:01.484Z to 192.168.1.124 (auth: True)
2019-04-30 01:41:07 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/calendars/calendar.XXXXXXXXXXXXgmailcom to 192.168.1.124 (auth: True)
2019-04-30 01:41:30 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:41:31.579Z to 192.168.1.124 (auth: True)
2019-04-30 01:42:08 DEBUG (MainThread) [homeassistant.components.http.view] Serving /auth/token to 10.8.0.6 (auth: False)
2019-04-30 01:42:08 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/websocket to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/slider-entity-row.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/card-tools.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/dark-sky-weather-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/secondaryinfo-entity-row.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/atomic-calendar.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/radar-forecast.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/tracker-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/entity-attributes-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/vertical-stack-in-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/tiles-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:09 DEBUG (MainThread) [homeassistant.components.http.view] Serving /customcards/lovelace/monster-card.js to 10.8.0.6 (auth: False)
2019-04-30 01:42:13 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:14.107Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:13 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:14.109Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:13 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:14.110Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:16 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:17.272Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:16 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:17.274Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:16 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:17.275Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:17 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-29T05:42:17.276Z to 10.8.0.6 (auth: True)
2019-04-30 01:42:19 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/calendars/calendar.XXXXXXXXXXXXgmailcom to 10.8.0.6 (auth: True)
2019-04-30 01:42:30 DEBUG (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-04-30T05:41:31.579Z to 192.168.1.124 (auth: True)
```

**Comment by ludeeus** [comment]
To be clear, this is not a bug, but just how it has to work.
I will look at alternatives.

**Comment by grantclem** [comment]
Lots of DEBUG entries also in the HA log - have manually changed line 53 from debug to warning. Sensor appears to have stopped working. Showing "Unknown" now. Is there some other workaround for this?

    logs = {'homeassistant.components.http.view': 'warning'}

**Comment by ludeeus** [comment]
No

**Comment by ludeeus** [comment]
Fixed in 1.0.0 <https://github.com/custom-components/authenticated/releases/tag/1.0.0>
